lib/scarpe/html.rb

```diff
@@ -32,7 +32,7 @@ def method_missing(name, *args, &block)
         result = block.call(self)
         @buffer += result if result.is_a?(String)
       else
-        @buffer += args.first
+        @buffer += args.first if args.first.is_a?(String)
```

Comment:
I think this has already been merged to main in a slightly different form.

Reply:
pushed, tests still pass